robot-server/robot_server/robot/calibration/deck/util.py

```diff
@@ -0,0 +1,25 @@
+from enum import Enum
+
+from http import HTTPStatus
+from robot_server.service.errors import RobotServerError
+from robot_server.service.json_api.errors import Error
+
+
+class DeckCalibrationError(Enum):
+    NO_PIPETTE = (
+        HTTPStatus.FORBIDDEN,
+        'No pipettes Attached',
+        'No pipettes present')
```

Comment:
```suggestion
        'At least one pipette must be attached to the OT-2 to run deck calibration')
```

This second field is the more detailed error explanation, which should always contain both a description of what is wrong and a suggested step to resolve the issue